Services/Simulations.cs

```diff
@@ -547,6 +552,11 @@ public async Task<bool> TryToSetDeviceDeletionCompleteAsync(string simulationId)
             simulation.DeviceCreationJobId = null;
             simulation.DevicesCreationStarted = false;
 
+            if (simulation.DeleteDevicesOnce == true)
+            {
+                simulation.DeleteDevicesOnce = false;
```

Comment:
don't need the if condition, just set the flag to it's default value i.e. false

Reply:
done